bangc-ops/mlu_op.h

```diff
@@ -10988,6 +10988,302 @@ mluOpDeformRoiPoolBackward(const mluOpHandle_t handle,
                            const mluOpTensorDescriptor_t grad_offset_desc,
                            void *grad_offset);
 
+// Group:BorderAlign
+/*!
+ * @brief Extracts the border features of \b input based on the bounding boxes to compute the
+ * maximum border features of \b input with the maximum pooling.
+ * The computing process of this operation is as follows:
+ *  1. For each border line of each box (commonly four lines: top, left, bottom and and right lines), uniformly samples
+ *     pool_size + 1 positions on this line, involving the start and end points.
+ *  2. Compute the corresponding features on these points by bilinear interpolation.
+ *  3. Perform the max pooling over all pool_size + 1 positions to output the pooled features.
+ *
+ * @param[in] handle
+ * Input. Handle to a Cambricon MLUOP context that is used to manage MLU devices and queues in
+ * ::mluOpBorderAlignForward operation. For detailed information, see ::mluOPHandle_t.
+ * @param[in] input_desc
+ * Input. The descriptor of the input tensor.
+ * @param[in] input
+ * Input. Pointer to the MLU memory that stores the input tensor. The shape of \b input is [N, H, W, 4C].
+ * Channels ranged in [0,C), [C,2C), [2C,3C), [3C,4C) represent the top, left, bottom and right features
+ * respectively.
+ * @param[in] boxes_desc
+ * Input. Descriptor of bounding box tensor.
+ * For detailed information, see ::mluOpTensorDescriptor_t.
+ * @param[in] boxes
+ * Input. Pointer to the MLU memory that stores boxes tensors. The shape of \b boxes is [N, H * W, 4].
+ * @param[in] pool_size
+ * Input. Number of positions sampled over the boxes' borders.
+ * @param[in] output_desc
+ * Input.  Descriptor of \b output, containing dimension and the layout of output.
+ * @param[out] output
+ * Output. Pointer to the MLU memory that stores the output tensor. The shape of
+ * argmax_idx is [N, H * W, 4, C].
+ * @param[in] argmax_idx_desc
+ * Input.  Descriptor of \b argmax_idx, containing dimension and the layout of \b argmax_idx.
+ * @param[out] argmax_idx
+ * Output. Pointer to the MLU memory that stores the \b argmax_idx tensor, which is the indices of
+ * maximum values after the max pooling. The shape of argmax_idx is [N, H * W, 4, C].
+ *
+ * @par Return
+ * - ::MLUOP_STATUS_SUCCESS, ::MLUOP_STATUS_BAD_PARAM.
+ *
+ * @par Formula
+ * - See "BorderAlignForward Operator" section in "Cambricon BANGC OPS User Guide" for details.
```

Comment:
BANG 和 C直接有空格【全文排查哦】
```suggestion
 * - See "BorderAlignForward Operator" section in "Cambricon BANG C OPS User Guide" for details.
```